charts/kam/values.yaml

```diff
@@ -56,6 +56,12 @@ backend:
         resources: [ "*" ]
         verbs: [ "get", "list", "create", "update", "delete", "patch", "watch" ]
 
+  env:
+    USE_JWT_TOKEN_PATHS: "false"
+    TOKEN_ROLE_PATHS: ""
+    TOKEN_PATHS_SEP: ","
+    TOKEN_PATH_SEGMENT_SEP: "."
+
```

Comment:
Dobrze rozumiem, że ta konfiguracja sprawi, że nie będziemy pobierali ról?

Reply:
Nope, ta konfiguracja sprawi, że będziemy pobierali role jak dotychczas na podstawie client and realm roles